.gitmodules

```diff
@@ -31,3 +31,6 @@
 [submodule "lib/sdl/SDL2_image"]
 	path = lib/sdl/SDL2_image
 	url = https://github.com/libsdl-org/SDL_image.git
+[submodule "lib/usb/ohci_usbhostlib"]
```

Comment:
*(This is a generic comment about PR description, but made on code so we can resolve it more easily)*

> 1. ~~Untested on 1.0 xboxes with daughterboard. I would expect this to have no issues though.~~ Has been tested ok!

So.. what kind of hardware has it been tested with? What hosts / what devices?
xpad in OpenXDK always had the issue that some 3rd party gamepads weren't supported, which was super frustrating.

Reply:
It was tested by some users in discord. It just testing the internal daughter which indeed just acts like a normal hub. 
Hub is connected to OHCI port 1:
Hub numbering is 3,4,2,1 as per other revs.

I have tested myself using downstream hubs and the internal daughterboard is just a commerical chip which nothing special. Without having a 1.0 myself, this is the most confident I can be.

>xpad in OpenXDK always had the issue that some 3rd party gamepads weren't supported, which was super frustrating.
Im not too sure how I can resolve this. As far as I can tell, I am doing it as correct as we understand it should be done. More exposure and users would help.

The only issue I can see is that I read/write to the appropriate interrupt pipes. Genuine controllers respect this and more inline with how the USB spec should be interpreted in my opinion, But you can also achieve this by doing control transfers. 3rd party may only respect control transfers. Im not sure what XDK does, I have a feeling it does control transfers for rumble atleast.

Reply:
>  Im not sure what XDK does, I have a feeling it does control transfers for rumble atleast.

We should be finding out asap and document in XboxDevWiki.
It's ridiculous that we still don't have these facts after almost 20 years.

Reply:
FTR, I've tested on my 1.0 using a Duke, controller S, a Logitech wheel ("Precision Vibration Feedback Wheel") and the Logitech wireless controller ("Cordless Precision Controller").
All worked great, rumble and all.